server/plugin.go

```diff
@@ -479,7 +489,7 @@ func (p *Plugin) onActivate() error {
 		Username:    botUsername,
 		DisplayName: botDisplayName,
 		Description: "Created by the MS Teams Sync plugin.",
-	}, pluginapi.ProfileImagePath("assets/icon.png"))
+	}, pluginapi.ProfileImageBytes(assets.Icon))
```

Comment:
`ProfileImagePath` had assumptions about the filesystem that were complicated by the reattach -- moving to embed and passing the bytes solves this.